stream-chat-android-client/src/androidTest/java/io/getstream/chat/android/client/poc/ClientInstrumentationTests.kt

```diff
@@ -3,9 +3,11 @@ package io.getstream.chat.android.client.poc
 import android.content.Context
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import androidx.test.platform.app.InstrumentationRegistry.getInstrumentation
+import io.getstream.chat.android.client.ChatClient
```

Comment:
These missing imports in the unused  `androidTest` source set broke my local test runs for some reason, so I added them